Highlight prerequisites line on amdgpu-install page

alexxu-amd opened the pull request:
Multiple feedbacks from internal and external mention the "set user group permission" command is not directly available from amdgpu-install page, which in fact, is included in the prerequisites page but tends to get ignored. Highlight the prerequisites line for improved visibility.

alexxu-amd committed:
highlight prerequisites block

yhuiYH commented on docs/install/amdgpu-install.rst:
```diff
@@ -22,7 +22,9 @@ Installation
 Installation of ``amdgpu-install`` differs slightly depending on the OS and its package manager.
 
-Make sure that the :doc:`/install/prerequisites` are met before installing.
+.. note::
```
```suggestion
.. important::
```

Do you want to try changing from "note" to "important"?  Do we want it to standout more?

alexxu-amd replied:
Yea, I agree. Wasn't aware of the important block

yhuiYH commented:
Also, to be complete, we should add a similar note on the "Ubuntu, SLES, and RHEL" install pages.  Can you add that to your PR?

alexxu-amd committed:
change note block to important block

alexxu-amd committed:
unifying native-install variants

alexxu-amd committed:
unify titles; add prereq important blocks for native install

alexxu-amd commented:
Added the notes for Ubuntu, SLES, and RHEL. Also added some minor changes to unify the content:
- RHEL refactors the installation part to install-rocm-template.rst, whereas the other two don't
- RHEL uses regular tense on titles, the other two uses present tense
- SLSE refers repo as "ROCm repositories", the other two just uses "repositories"

alexxu-amd committed:
remove install-rocm-template.rst

yhuiYH commented on docs/install/native-install/ubuntu.rst:
```diff
@@ -86,6 +86,10 @@ Add the ROCm repository.
 Installing
 ================================================
 
+.. important::
```
@dbieleck Do you like where @alexxu-amd added it in the "Installing" part?  Or do you think it should be right at the top of the page?

I think I'd like it right up at line 8, as the first thing they see, even before they start "registering the repositories".

dbieleck replied:
I agree.  It might be better up at line 8 so it's visible right away.

alexxu-amd committed:
bring the note to the top

yhuiYH reviewed (approved):
OK, this LGTM now.  Approved.
